crates/moss-theme/theme_util.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+from pathlib import Path
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description="Generate Rust Constants for JSON Schema ")
+    parser.add_argument("--schema", type=argparse.FileType('r'), required=True)
+    parser.add_argument("--output", type=argparse.FileType('w+'), required=True)
+    args = parser.parse_args()
+
+    schema_file = args.schema
+    output_file = args.output
+
+    schema = schema_file.read()
```

Comment:
After reading the file, we must ensure it is valid and that we are not generating invalid Rust code.

```python 
def validate_json(schema: str) -> dict:
    """
    Validate that the schema string is valid JSON.
    Returns the parsed JSON object if valid.
    Raises ValueError if the JSON is invalid.
    """
    try:
        return json.loads(schema)
    except json.JSONDecodeError as e:
        raise ValueError(f"Invalid JSON: {e}")
```